### PR TITLE
add docs for support for roles, cluster roles and subjects in CLI

### LINKED ISSUE #505: [1.7.0] Document updates to `kyverno-test.yaml` to cover Roles, ClusterRoles, and Subjects
### Description

Based on PR https://github.com/kyverno/kyverno/pull/3188 we need to update docs to cover the user info section.

### Slack discussion

_No response_

### PR DESCRIPTION
**Pull request by 4molybdenum2**
Signed-off-by: Tathagata Paul <tathagatapaul7@gmail.com>

## Related issue #
Closes #505 

## Proposed Changes
- [x] Add docs for kyverno apply command for support in roles, cluster roles and subjects.
- [x] Add docs for kyverno test command for support in roles, cluster roles and subjects.

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [ ] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Comment by 4molybdenum2** [comment]
@chipzoller Review needed. Also, I think few of the flags can be better named, which I will change shortly... (ex: userinfo -> user-info)

**Comment by chipzoller** [comment]
`user_info.yaml` is not a hard-coded name, right?

**Comment by 4molybdenum2** [comment]
Yes

File: content/en/docs/Kyverno CLI/_index.md [hunk 1]
```diff
@@ -115,12 +115,14 @@ kyverno apply /path/to/policy.yaml --resource /path/to/resource.yaml --set <vari
 
 Use `-f` or `--values-file` for applying multiple policies to multiple resources while passing a file containing variables and their values. Variables specified can be of various types include AdmissionReview fields, ConfigMap context data (Kyverno 1.3.6), and API call context data (Kyverno 1.3.6).
 
+Use `-u` or `--userinfo` for applying policies while passing a optional user_info.yaml file which contains necessary admission request data made during the request.
```
Comment (chipzoller) [inline content/en/docs/Kyverno CLI/_index.md:118]: "while passing an optional..."

Comment (chipzoller) [reply content/en/docs/Kyverno CLI/_index.md:118]: A question on your code...the file name is not hard-coded to look for `user_info.yaml`, is it?

Comment (4molybdenum2) [reply content/en/docs/Kyverno CLI/_index.md:118]: The file name is not hardcoded. The user can pass a cmd flag ```userInfoPath``` to set the file path

**Pull request merged**